packages/plugins/src/plugins/renzo/helpers.ts

```diff
@@ -4,34 +4,80 @@ import { getEvmClient } from '../../utils/clients';
 import {
   activeStakeAbi,
   getOutstandingWithdrawRequestsAbi,
+  getRateAbi,
   withdrawRequestAbi,
 } from './abis';
 import { Address } from 'viem';
 import { getBalances } from '../../utils/evm/getBalances';
 import { MulticallIO } from '../../utils/octav/types/multicallIO';
 import { zeroBigInt } from '../../utils/misc/constants';
 
-export type ActiveStakeIO = MulticallIO<typeof activeStakeAbi, 'activeStake'>;
-export type OutstandingWithdrawRequestsIO = MulticallIO<
+type ActiveStakeIO = MulticallIO<typeof activeStakeAbi, 'activeStake'>;
+type OutstandingWithdrawRequestsIO = MulticallIO<
   typeof getOutstandingWithdrawRequestsAbi,
   'getOutstandingWithdrawRequests'
 >;
+type GetRateIO = MulticallIO<typeof getRateAbi, 'getRate'>;
+
+interface BalanceRate {
+  contract: RenzoStakedContractConfig;
+  balance: bigint | null;
```

Comment:
Can we use `undefined` instead of `null` ?
```suggestion
  balance: bigint | undefined;
```

That would bring more consistency with the `rate` attribute just below.

Reply:
null is coming from getBalances, a function already written by sonarwatch. I am using null cause that's what it returns